js/client-topbar.js

```diff
@@ -1060,4 +1070,25 @@
 		}
 	});
 
+	var MutePopup = this.MutePopup = Popup.extend({
```

Comment:
I would call this something else; it sounds like a popup you get when staff mute you.